src/test/unit/controls/Gantt/GanttGenerate-spec.js

```diff
@@ -1141,18 +1141,18 @@ describe("Gantt - Generate", () => {
                 done();
             });
         });
-        it("creates multiple datelines correctly", () => {
+        it("creates multiple eventlines correctly", () => {
```

Comment:
Create another file similar to dateline and add these and `Pass Through` tests.

Reply:
`GraphEventline-spec.js`

Reply:
For Graph I have already created GraphEventline-spec.js do we need to create one for Gantt too (GanttEventline-spec.js).

Reply:
Yes please